src/resources/dicts/de.edn

```diff
@@ -299,6 +299,7 @@
  :settings-page/custom-configuration "Individuelle Konfiguration"
  :settings-page/custom-date-format "Bevorzugtes Datumsformat"
  :settings-page/custom-date-format-warning "Neuindizierung erforderlich! Vorhandene Journal-Verweise würden nicht mehr funktionieren!"
+ :settings-page/preferred-pasting-file-hint "When enabled, pasting an image from the internet will download and insert the image. When disabled, it will paste the link to the image."
```

Comment:
Worth removing all the non-english English entries like this. A language file should only contain entries for its language. Our translation system already handles defaulting to English when a key doesn't exist in another language

Reply:
@logseq-cldwalker sounds good will remove it. The reason I thought it should be included is to make it easier for translators to know what still needs to be translated.

Reply:
Gotcha. Most translators rely on [this workflow](https://github.com/logseq/logseq/blob/master/docs/contributing-to-translations.md#edit-a-language) which depends on entries being missing in a language

We should get rid of English entries in any non-English file as they are redundant and confusing translators. I'll try to knock this out next week but PR is welcome for this as well